packages/tupaia-web/src/features/Visuals/View/transformDownloadLink.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+export const transformDownloadLink = (resourceUrl: string) => {
+  const baseUrl = import.meta.env.REACT_APP_TUPAIA_WEB_API_URL || 'http://localhost:8080/api/v1/';
```

Comment:
I think this is the same logic as is in api.ts. We could move that to a constants file or even just export it and use it from the api file since it's the api url.

Reply:
Also could you please add a comment to the util, I'm not 100% sure what it is doing?